lib/pypy.js

```diff
@@ -23,8 +23,8 @@ if (typeof module !== "undefined") {
 var debug = function(){};
 if (typeof console !== "undefined") {
   debug = console.log;
```

Comment:
You might need to change this to:

    debug = console.log.bind(console);

Otherwise, doing this in either chrome or firefox will throw a `TypeError`. It seems to work fine in node though.